### ISSUE TITLE
GetOrPost parameters added to the URL query for a PUT request

### ISSUE THREAD

**Issue by tcorrin** [FubarDevelopment/restsharp.portable#98]
Get Or Post parameters are added as `x-www-form-urlencoded` for a POST request or in the URL query for a GET request however for a PUT request they are added using both methods. The `x-www-form-urlencoded` check is `!= GET` and the query string check is `!= POST`.

**Comment by jchannon** [comment]
This appears to be:

https://github.com/VQComms/restsharp.portable/blob/master/src/FubarCoder.RestSharp.Portable.HttpClient/RestClientExtensions.cs#L88 = `!= GET add params to body`

https://github.com/FubarDevelopment/restsharp.portable/blob/6811f9290f83d243a1cbc976843e2ba473015fa1/src/FubarCoder.RestSharp.Portable.Core/RestClientExtensions.cs#L163 `!= POST add param to querystring`

As @tcorrin says, both should be != GET

**Comment by fubar-coder** [comment]
Are you sure that this should be `!= GET`? This condition determines the parameters added to the query string and the `!= POST` ensures that `GetOrPost` parameters are added to the query string for all non-`POST` requests.

**Comment by jchannon** [comment]
It should be != POST && != PUT I think

**Comment by HappyTreesDev** [comment]
Yes, this causes huge issues when I attempt to send a file through a put.
